src/main/java/mate/academy/intro/model/Book.java

```diff
@@ -37,6 +45,16 @@ public class Book {
 
     private String description;
 
+    @ManyToMany(fetch = FetchType.LAZY)
```

Comment:
```suggestion
    @ManyToMany 
```
It’s  default fetch type